browser/js/identities.js

```diff
@@ -1,20 +1,25 @@
 // All of our identity services and matchers are defined here.
 
+// parseLocationQuery converts URL-encoded parameters into an object. It
+// requires unique keys, will throw an error if there is a duplicate key.
 function parseLocationQuery(s) {
```

Comment:
Reformatted here, [use ?w=1 for a cleaner diff](https://github.com/keybase/client/pull/10277/commits/3edc650ea3a9fa789d239d84ea2cab3ea7780e54?w=1).